java-tests/vanilla/build.gradle.kts

```diff
@@ -65,3 +48,13 @@ dependencies {
     testImplementation(spine.base)
     testImplementation(spine.testlib)
 }
+
+sourceSets {
+    all {
+        java.setSrcDirs(java.filter { !it.absolutePath.startsWith("${buildDir.absolutePath}/generated") })
```

Comment:
This piece will seem odd in a few months, or so. Could you please document the idea? I.e. why would we need to do that at all?